index.go

```diff
@@ -394,10 +394,9 @@ func (i *Index) createFrame(name string, opt FrameOptions) (*Frame, error) {
 	if opt.RangeEnabled {
```

Comment:
let's remove the three comment lines above this (talking about issue 399 and "none")